example/App.js

```diff
@@ -8,7 +8,14 @@
  */
 
 import React, {Component} from 'react';
-import {ScrollView, StyleSheet, Text, SafeAreaView} from 'react-native';
```

Comment:
This App.js file is the only one that is copied to safety during the example refresh run IIRC, so this will actually persist. But to the extent it deviates from the stock App.js from the template, deviations should be minimized - nothing with regard to style or whitespace or anything has value - only demonstrating functionality